### ISSUE TITLE
Cannot find method handle with parameter types: [org.eclipse.jetty.server.HttpChannel]

### ISSUE THREAD

**Issue by dawidmalina** [naver/pinpoint#1257]
jetty 8.1.15.v20140411
spring 4.1.6
```
2015-11-25 20:40:13 [ERROR](c.n.p.p.ClassFileTransformerDispatcher:125) Transformer:com.navercorp.pinpoint.profiler.plugin.MatchableClassFileTransformerGuardDelegate threw an exception. cl:sun.misc.Laun
cher$AppClassLoader@14dad5dc ctxCl:sun.misc.Launcher$AppClassLoader@14dad5dc agentCl:com.navercorp.pinpoint.bootstrap.PinpointURLClassLoader@39ba5a14 Cause:com.navercorp.pinpoint.bootstrap.instrument.No
tFoundInstrumentException: Cannot find method handle with parameter types: [org.eclipse.jetty.server.HttpChannel]
com.navercorp.pinpoint.exception.PinpointException: com.navercorp.pinpoint.bootstrap.instrument.NotFoundInstrumentException: Cannot find method handle with parameter types: [org.eclipse.jetty.server.Htt
pChannel]
        at com.navercorp.pinpoint.profiler.plugin.MatchableClassFileTransformerGuardDelegate.transform(MatchableClassFileTransformerGuardDelegate.java:72)
        at com.navercorp.pinpoint.profiler.ClassFileTransformerDispatcher.transform0(ClassFileTransformerDispatcher.java:119)
        at com.navercorp.pinpoint.profiler.ClassFileTransformerDispatcher.transform(ClassFileTransformerDispatcher.java:100)
        at sun.instrument.TransformerManager.transform(TransformerManager.java:188)
        at sun.instrument.InstrumentationImpl.transform(InstrumentationImpl.java:428)
        at java.lang.ClassLoader.defineClass1(Native Method)
        at java.lang.ClassLoader.defineClass(ClassLoader.java:760)
        at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142)
        at java.net.URLClassLoader.defineClass(URLClassLoader.java:467)
        at java.net.URLClassLoader.access$100(URLClassLoader.java:73)
        at java.net.URLClassLoader$1.run(URLClassLoader.java:368)
        at java.net.URLClassLoader$1.run(URLClassLoader.java:362)
        at java.security.AccessController.doPrivileged(Native Method)
        at java.net.URLClassLoader.findClass(URLClassLoader.java:361)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
        at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
        at org.apache.cxf.transport.http_jetty.JettyHTTPServerEngine.addServant(JettyHTTPServerEngine.java:327)
        at org.apache.cxf.transport.http_jetty.JettyHTTPDestination.activate(JettyHTTPDestination.java:179)
        at org.apache.cxf.transport.AbstractObservable.setMessageObserver(AbstractObservable.java:53)
        at org.apache.cxf.binding.AbstractBindingFactory.addListener(AbstractBindingFactory.java:95)
        at org.apache.cxf.jaxrs.JAXRSBindingFactory.addListener(JAXRSBindingFactory.java:88)
        at org.apache.cxf.endpoint.ServerImpl.start(ServerImpl.java:123)
        at org.apache.cxf.jaxrs.JAXRSServerFactoryBean.create(JAXRSServerFactoryBean.java:206)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:497)
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeCustomInitMethod(AbstractAutowireCapableBeanFactory.java:1702)
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1641)
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1570)
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:539)
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:476)
        at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:303)
        at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230)
        at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:299)
        at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:194)
        at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:755)
        at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:757)
        at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:480)
        at com.company.BaseApp.run(BaseApp.java:46)
        at com.company.mock.app.MockApp.main(MockApp.java:13)
orp.pinpoint.profiler.plugin.MatchableClassFileTransformerGuardDelegate
```

```
2015-11-25 16:48:49 [ERROR](c.n.p.p.ClassFileTransformerDispatcher:125) Transformer:com.navercorp.pinpoint.profiler.plugin.MatchableClassFileTransformerGuardDelegate threw an exception. cl:sun.misc.Laun
cher$AppClassLoader@14dad5dc ctxCl:sun.misc.Launcher$AppClassLoader@14dad5dc agentCl:com.navercorp.pinpoint.bootstrap.PinpointURLClassLoader@39ba5a14 Cause:com.navercorp.pinpoint.bootstrap.instrument.No
tFoundInstrumentException: Cannot find method handle with parameter types: [org.eclipse.jetty.server.HttpChannel]
com.navercorp.pinpoint.exception.PinpointException: com.navercorp.pinpoint.bootstrap.instrument.NotFoundInstrumentException: Cannot find method handle with parameter types: [org.eclipse.jetty.server.Htt
pChannel]
        at com.navercorp.pinpoint.profiler.plugin.MatchableClassFileTransformerGuardDelegate.transform(MatchableClassFileTransformerGuardDelegate.java:72)
        at com.navercorp.pinpoint.profiler.ClassFileTransformerDispatcher.transform0(ClassFileTransformerDispatcher.java:119)
        at com.navercorp.pinpoint.profiler.ClassFileTransformerDispatcher.transform(ClassFileTransformerDispatcher.java:100)
        at sun.instrument.TransformerManager.transform(TransformerManager.java:188)
        at sun.instrument.InstrumentationImpl.transform(InstrumentationImpl.java:428)
        at java.lang.ClassLoader.defineClass1(Native Method)
        at java.lang.ClassLoader.defineClass(ClassLoader.java:760)
        at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142)
        at java.net.URLClassLoader.defineClass(URLClassLoader.java:467)
        at java.net.URLClassLoader.access$100(URLClassLoader.java:73)
        at java.net.URLClassLoader$1.run(URLClassLoader.java:368)
        at java.net.URLClassLoader$1.run(URLClassLoader.java:362)
        at java.security.AccessController.doPrivileged(Native Method)
        at java.net.URLClassLoader.findClass(URLClassLoader.java:361)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
        at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
        at org.apache.cxf.transport.http_jetty.JettyHTTPServerEngine.addServant(JettyHTTPServerEngine.java:327)
        at org.apache.cxf.transport.http_jetty.JettyHTTPDestination.activate(JettyHTTPDestination.java:179)
        at org.apache.cxf.transport.AbstractObservable.setMessageObserver(AbstractObservable.java:53)
        at org.apache.cxf.binding.AbstractBindingFactory.addListener(AbstractBindingFactory.java:95)
        at org.apache.cxf.jaxrs.JAXRSBindingFactory.addListener(JAXRSBindingFactory.java:88)
        at org.apache.cxf.endpoint.ServerImpl.start(ServerImpl.java:123)
        at org.apache.cxf.jaxrs.JAXRSServerFactoryBean.create(JAXRSServerFactoryBean.java:206)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:497)
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeCustomInitMethod(AbstractAutowireCapableBeanFactory.java:1702)
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1641)
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1570)
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:539)
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:476)
        at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:303)
        at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230)
        at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:299)
        at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:194)
        at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:755)
        at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:757)
        at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:480)
        at com.company.BaseApp.run(BaseApp.java:46)
        at com.company.mock.app.MockApp.main(MockApp.java:13)
Caused by: com.navercorp.pinpoint.bootstrap.instrument.NotFoundInstrumentException: Cannot find method handle with parameter types: [org.eclipse.jetty.server.HttpChannel]
        at com.navercorp.pinpoint.profiler.instrument.JavassistClass.addInterceptor0(JavassistClass.java:528)
        at com.navercorp.pinpoint.profiler.instrument.JavassistClass.addInterceptor0(JavassistClass.java:487)
        at com.navercorp.pinpoint.profiler.instrument.JavassistClass.addInterceptor(JavassistClass.java:400)
        at com.navercorp.pinpoint.bootstrap.instrument.transformer.TransformUtils$1.doInTransform(TransformUtils.java:37)
        at com.navercorp.pinpoint.profiler.plugin.MatchableClassFileTransformerGuardDelegate.transform(MatchableClassFileTransformerGuardDelegate.java:70)
        ... 41 more
```

**Comment by koo-taejin** [comment]
I saw the jetty plugin code. 
It focus only 9 version. 

I'll see if I can support.

**Comment by koo-taejin** [comment]
@dawidmalina 
could you check it please!

**Comment by dawidmalina** [comment]
Thx for immediate response. I will try to build and test in next week.

**Comment by dawidmalina** [comment]
BTW Do you know when 1.5.1 will be released?

**Comment by koo-taejin** [comment]
1.5.1 will be released on next week. 
It is made up of mostly to fix bug.

**Comment by emeroad** [comment]
Fixed 1.5.1
